pkg/version/release.go

```diff
@@ -25,22 +27,25 @@ func ParseGitVersion(gitVersion string) (*ReleaseVersion, error) {
 	}, nil
 }
 
-// FirstMinorRelease returns the minor release but the patch releases always be 0(vx.y.0). e.g:
-// - v1.2.1-12-g2eb92858 --> v1.2.0
-// - v1.2.3-12-g2e860210 --> v1.2.0
-func (r *ReleaseVersion) FirstMinorRelease() string {
+// ReleaseVersion returns the current version with format "vx.y.z".
```

Comment:
This function not only returns the `vx.y.z` anymore, but including a pre-releases format.
Please update the comments as well.